<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="KALAMSIR-PC" name="tests.Runner2" tests="16" failures="7" timestamp="2022-08-27T12:54:09 IST" time="13.680" errors="0">
  <testcase name="runScenario" time="0.502" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="2.926" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at gluecode.StepDefs5.method3(StepDefs5.java:112)
at ✽.validate response as per data in excel file(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature5.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.011" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.524" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "200" as per "duplicate" and content type is "application/json"(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.530" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs4.method2(StepDefs4.java:40)
at ✽.status code is 204(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature4.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.058" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.610" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "400" as per "wrong" and content type is "application/json"(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="1.569" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at gluecode.StepDefs3.method1(StepDefs3.java:77)
at ✽.validate that restful service with content type as "application/json"(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature3.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.544" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.512" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="5.197" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false],
	expected [true] but found [false],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false],
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at gluecode.StepDefs4.method13(StepDefs4.java:93)
at ✽.validate response as per data in text file(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature4.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.529" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.009" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.055" classname="tests.Runner2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at gluecode.StepDefs2.method1(StepDefs2.java:43)
at ✽.status code is "400" as per "blank" in json response body(file:///E:/batch259/org.magnitia.restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.054" classname="tests.Runner2"/>
  <system-out/>
  <testcase name="runScenario" time="0.050" classname="tests.Runner2"/>
  <system-out/>
</testsuite> <!-- tests.Runner2 -->
